pages/apply/confirmation.tsx

```diff
@@ -32,6 +35,7 @@ const ApplicationConfirmation = (): JSX.Element => {
       <Panel
         heading="Application complete"
         message={`Your reference number: ${application.reference?.toUpperCase()}`}
+        email={residentEmail}
```

Comment:
Set the text here, rather than within the panel.

Reply:
That's done now